backend/queries/events.js

```diff
@@ -7,65 +7,98 @@ Events Route Queries | Capstone App (Pursuit Volunteer Mgr)
 const db = require('../db/db');
 
 
+/* HELPER FUNCTION */
+const formatStr = str => {
+  return str.toLowerCase()
+}
+
 /* QUERIES */
 
 
 // Get all events (past events are auto pushed to the back)
-const getAllEvents = async () => {
+const getAllEvents = async (vName, topic, instructor, upcoming, past) => {
+
```

Comment:
it's possible to avoid the code lines 21 to 33 by dong lowerCase on the call

Reply:
👍🏽